static/app/components/searchQueryBuilder/tokens/useSortedFilterKeyItems.tsx

```diff
@@ -149,6 +150,9 @@ export function useSortedFilterKeyItems({
   const hasRawSearchReplacement = organization.features.includes(
     'search-query-builder-raw-search-replacement'
   );
+  const hasMatchKeySuggestions = organization.features.includes(
```

Comment:
Instead of putting this in the search component, it might make more sense to put it in the TraceItemSearchQueryBuilder to decide whether or not to pass this prop. Otherwise you might confuse people wondering why the prop doesn't work

Reply:
I'll make a quick change, and do this for the raw search replacement as well, and throw in stories for both.